content-src/components/TopSites/TopSites.js

```diff
@@ -46,13 +46,13 @@ const TopSitesItem = React.createClass({
     return faviconSize;
   },
   userEvent(event) {
-    const {page, source, index, dispatch, site} = this.props;
+    const {page, source, index, dispatch, metadata_source} = this.props;
     let payload = {
       event,
       page,
       source,
       action_position: index,
-      metadata_source: site.metadata_source
```

Comment:
was actually a bug. because metadata_Source is on `props` and not `props.site`. but it's in a separate commit